packages/@react-spectrum/table/src/useTableColumnResizeState.ts

```diff
@@ -139,20 +150,16 @@ export function useTableColumnResizeState<T>(props: ColumnResizeStateProps<T>):
     affectedColumnWidthsRef.current = [];
 
     let widths = new Map<Key, number>(columnWidthsRef.current);
-    // Need to set the resizeBufferColumn or "spooky column" back to 0 since done resizing;
-    const bufferColumnKey = columnsRef.current[columnsRef.current.length - 1].key;
-    widths.set(bufferColumnKey, 0);
```

Comment:
I'll probably ask for a explanation next week on how you were able to get rid of the spooky column

Reply:
honestly, i'm pretty sure it's because we fixed the virtualizer behavior and stopped generating so many TableLayouts. all I had to do was delete the lines that referred to it and it worked as I expected it to

Reply:
I'll be running it past Danny and Marshall though, just to make sure I'm not missing anything